source/game/scene/GameScene.cc

```diff
@@ -59,6 +86,7 @@ void GameScene::initScene() {
     createEngines();
     System::KPadDirector::Instance()->reset();
     initEngines();
+    checkMemory();
```

Comment:
Is it really worth checking this in non-debug? This is adding unnecessary computation on **every frame** in release.

Reply:
Not every frame. Memory checking is only done during scene initialization / scene destruction (because we know the heap is locked during the race itself).

Reply:
Oh sorry Github made it look like this was in `calc`